src/main/scala/com/danielasfregola/twitter4s/entities/User.scala

```diff
@@ -46,6 +47,7 @@ final case class User(blocked_by: Boolean = false,
                 status: Option[Tweet] = None,
                 statuses_count: Int,
                 time_zone: Option[String] = None,
+                translator_type: Option[String] = None,
```

Comment:
I searched in the tweeter doc if maybe they provide a list of possible values for translator type, but couldn't find any. So I think using String rather than a dedicated enough is the right call